forge-gui/res/cardsfolder/upcoming/kozileks_unsealing.txt

```diff
@@ -0,0 +1,9 @@
+Name:Kozilek's Unsealing
+ManaCost:2 U
+Types:Enchantment
+K:Devoid
+T:Mode$ SpellCast | ValidCard$ Creature.cmcEQ4,Creature.cmcEQ5,Creature.cmcEQ6 | ValidActivatingPlayer$ You | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a creature spell with mana value 4, 5, or 6, create two 0/1 colorless Eldrazi Spawn creature tokens with "Sacrifice this creature: Add {C}."
+T:Mode$ SpellCast | ValidCard$ Creature.cmcGE7 | ValidActivatingPlayer$ You | Execute$ TrigDraw | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a creature spell with mana value 7 or greater, draw three cards.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 3
+SVar:TrigToken:DB$ Token | TokenAmount$ 2 | TokenScript$ c_0_1_eldrazi_spawn_sac | TokenOwner$ You
```

Comment:
imo move the `TrigToken` line in between the Line 5-6 after its trigger

Reply:
Done